lib/ansible/module_utils/redfish_utils.py

```diff
@@ -0,0 +1,932 @@
+        data = response['data']
+        if 'Systems' not in data:
+            return {'ret': False, 'msg': "Systems resource not found"}
+        else:
+            systems = data["Systems"]["@odata.id"]
+            response = self.get_request(self.root_uri + systems)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            for member in data[u'Members']:
+                systems_service = member[u'@odata.id']
+            self.systems_uri = systems_service
+            return {'ret': True}
+
+    def _find_updateservice_resource(self, uri):
+        response = self.get_request(self.root_uri + uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        if 'UpdateService' not in data:
+            return {'ret': False, 'msg': "UpdateService resource not found"}
+        else:
+            update = data["UpdateService"]["@odata.id"]
+            self.update_uri = update
+            response = self.get_request(self.root_uri + update)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            firmware_inventory = data['FirmwareInventory'][u'@odata.id']
+            self.firmware_uri = firmware_inventory
+            return {'ret': True}
+
+    def _find_chassis_resource(self, uri):
+        chassis_service = []
+        response = self.get_request(self.root_uri + uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        if 'Chassis' not in data:
+            return {'ret': False, 'msg': "Chassis resource not found"}
+        else:
+            chassis = data["Chassis"]["@odata.id"]
+            response = self.get_request(self.root_uri + chassis)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            for member in data[u'Members']:
+                chassis_service.append(member[u'@odata.id'])
+            self.chassis_uri_list = chassis_service
+            return {'ret': True}
+
+    def _find_managers_resource(self, uri):
+        response = self.get_request(self.root_uri + uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        if 'Managers' not in data:
+            return {'ret': False, 'msg': "Manager resource not found"}
+        else:
+            manager = data["Managers"]["@odata.id"]
+            response = self.get_request(self.root_uri + manager)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            for member in data[u'Members']:
+                manager_service = member[u'@odata.id']
+            self.manager_uri = manager_service
+            return {'ret': True}
+
+    def get_logs(self):
+        log_svcs_uri_list = []
+        list_of_logs = []
+        result = {}
```

Comment:
Should this be `response` since that is what is returned? I don't see `result` used within this function.